libs/explored-types/src/api.ts

```diff
@@ -8,6 +8,7 @@ import {
   ConsensusNetwork,
   ConsensusState,
   Currency,
+  Event,
```

Comment:
Should this be ExplorerEvent if its not the Event from our core `types` package?